CONTRIBUTING.md

```diff
@@ -175,10 +175,12 @@ Project maintainers are responsible for releasing new versions of pycytominer.
 Creating a new release includes the following steps:
 
 1. Create a new branch from `main` for the release (e.g. `release-v1.0.0`)
-2. Run the command `cz bump --files-only` to update the version number in `CITATION.cff` and `pyproject.toml:tool.commitizen`.
-3. Review the changes to CHANGELOG.md and edit as needed.
-4. Review the [commit history](https://github.com/cytomining/pycytominer/compare) from the last release and add documentation for changes that weren't auto-included because they didn't follow the conventional-commit standard.
-5. `git add` all changes and run `cz bump` to create the release commit.
+2. Review the [commit history](https://github.com/cytomining/pycytominer/compare) from the last release and check whether it includes commits that don't follow the conventional commit standard.
+   If all changes follow conventional commits, skip to step 5.
+3. Run the command `cz bump --files-only` to update the version number in `CITATION.cff` and `pyproject.toml:tool.commitizen` and generate the draft changelog.
```

Comment:
Would it make sense to add a "pre-step" or a link here to help describe where the `cz` command-line tool may be installed from?